shared/src/common/types/groups/groups-update-request-dto.ts

```diff
@@ -0,0 +1,8 @@
+type GroupsUpdateRequestDto = {
+  id: number;
+  name?: string;
+  permissionIds?: number[];
+  userIds?: number[];
```

Comment:
Payload to update fields should be obligatory
```suggestion
  name: string;
  permissionIds: number[];
  userIds: number[];
```